source/guide_maildrop-autoresponder.rst

```diff
@@ -0,0 +1,260 @@
+
+The stored email can be both plaintext and HTML. If there's no mail stored in the specified folder mailbot won't do anything. The only downside of this solution to be mentioned: mailbot takes the incoming mail and will add it as an attachment to the replied message. Unfortunately there is no way to disable this behaviour.
+
+Installation
+------------
+
+In the ``~/autoresponder/.autoreply-filter`` the only thing that changes is the ``cc`` command. 
+
+Use your favourite editor to edit ``~/autoresponder/.autoreply-filter``. Remove these lines
+
+.. code-block:: ini
+
+	# reply to mail with the following text and given parameters
+	cc "| mailbot -t $HOME/autoresponder/autoresponse.txt -N -A 'From: $FROM' -d $HOME/autoresponder/bouncedb -D 3 /var/qmail/bin/qmail-inject -f ''"
+
+and replace them with the following lines:
+
+	# using the mail stored in the referenced IMAP-Folder to reply
+	cc "| mailbot -T replydraft -l '$MAILDIR/.Autoresponder' -N  -A 'From: $FROM' -d $HOME/autoresponder/bouncedb -D 3 /var/qmail/bin/qmail-inject -f ''"
+	
+You can customize the ``cc``-command to your needs:
+
+Configuration
+------------
+
+``mailbot`` requires some additional parameters:
+
+  * ``-T replydraft``: Specifies the type of reply
+  * ``-l '$MAILDIR/.Autoresponder'``: Specifies the ``IMAP``-Folder in the ``$MAILDIR`` (aka. mailbox). Mailbox will use the most recent message in this folder and reply it to the sender
+  * ``-N``: Disables quoting the original message
+  * ``-A 'From: $FROM'``: Adds a header to the response. In our case, we'll set the ``From:`` header in the autogenerated response.
+  * ``-d $HOME/autoresponses/bouncedb``: Creates a small database ``bouncedb`` in the specified path, that keeps track of senders' E-mail addresses, and prevent duplicate autoresponses going to the same address. Another autoresponse to the same address will not be mailed until at least the amount of time specified by the -D option has elapsed.
+  * ``-D 3``: Do not send duplicate autoresponses (see the -d flag) for at least ``3`` days (default: 1 day).
+  * ``/var/qmail/bin/qmail-inject -f ''``: utility to send the mail	
+  
+.. note :: ``-N`` is useless when using ``-T replydraft``. Unfortunately there is no way to disable this behaviour. Nevertheless is set to prevent syntax issues.
+
+Since we refer to the ``IMAP``-Folder ``Autorespond`` in the users mailbox tell them to create them and store a message there. Use your favorite IMAP-Client to create & save a new draft. Afterwards move the draft to the ``Autoresponder`` folder. Its not necessary to enter any recipients, mailbot will delete them anyway. Now you're done!
+
+.. note :: You can choose a different name for the IMAP-Folder. Make sure to change the reference in the ``~/.autoreply-filter`` file then accordingly.
+
+Usage
+-----
+
+When you're using the IMAP method you can additionally disable or enable the autoresponder by removing or adding a mail in the specified ``IMAP``-Folder. 
```

Comment:
If I understand the mailbot documentation correctly, it will always use the most recent draft present in that folder. Might be worth mentioning.